.changeset/changelog-lit.cjs

```diff
@@ -101,16 +99,41 @@ const changelogFunctions = {
 
     const users = usersFromSummary.length
       ? usersFromSummary
+          .filter((u) => !containsLitTeamMemberUsername(u))
           .map(
             (userFromSummary) =>
               `[@${userFromSummary}](https://github.com/${userFromSummary})`
           )
           .join(', ')
+      : containsLitTeamMemberUsername(links.user)
+      ? null
       : links.user;
 
+    /**
+     * containsLitTeamMemberUsername lets us only congratulate community
+     * contributions.
+     * @param {string} s - any string that may contain a username.
+     * @returns boolean indicating if a Lit team member was found in the string.
+     */
+    function containsLitTeamMemberUsername(s) {
+      return [
+        'AndrewJakubowicz',
+        'augustjk',
+        'bicknellr',
+        'dfreedm',
+        'e111077',
+        'justinfagnani',
+        'kevinpschaaf',
+        'rictic',
+        'sorvell',
+        'usergenic',
+      ].some((coreTeamUser) => s.includes(coreTeamUser));
```

Comment:
```ts
return new RegExp(`\\b(${[
        'AndrewJakubowicz',
        'augustjk',
        'bicknellr',
        'dfreedm',
        'e111077',
        'justinfagnani',
        'kevinpschaaf',
        'rictic',
        'sorvell',
        'usergenic',
      ].join('|')})\\b`, 'i').test(s);
```

I went down a rabbit-hole and overthought the problem of usernames that might be supersets of our usernames :-P

LGTM tho.

Reply:
hahaha that's a great point. impersonators don't deserve thanks!

Reply:
Applied and tested this fantastic solution! Thank you! We definitely want to thank `rictickler` or `chadfreeman` if they contribute!